miden-tx/src/testing/tx_context/builder.rs

```diff
@@ -227,7 +227,7 @@ impl TransactionContextBuilder {
                 call.wallet::create_note
 
                 push.{asset}
-                call.{ACCOUNT_ADD_ASSET_TO_NOTE_MAST_ROOT}
+                call.account::add_asset_to_note
```

Comment:
We now have `miden::contracts::wallets::basic::move_asset_to_note` - should we use it here instead?

Reply:
If we do `move_asset_to_note`, it fails with an underflow for that asset. I believe it's because the account does not contain it.

Reply:
I see. So here we just want to test that adding an asset to note works, and we don't care about asset consistency, right?

Reply:
Correct. There's some other tests that test asset consistency that do use `move_asset_to_note` though.